src/iframe-worker.js

```diff
@@ -62,12 +64,16 @@ class IframeWorker extends EventTarget {
   }
 
   postMessage(msg) {
-    this.iframe.contentWindow.postMessage(msg, '*')
+    if (this.iframe) {
+      this.iframe.contentWindow.postMessage(msg, '*')
+    }
   }
 
   terminate() {
-    window.removeListener('message', this.handleIframeMessage)
-    document.removeChild(this.iframe)
+    window.removeEventListener('message', this.handleIframeMessage)
+    if (this.iframe) {
+      document.removeChild(this.iframe)
```

Comment:
Shouldn’t `removeChild` be called on the parent node?

```jsx
      this.iframe.parentNode.removeChild(this.iframe)
```

But I think we can safely use [`.remove()`](https://developer.mozilla.org/en-US/docs/Web/API/ChildNode/remove) since we don’t support IE:

```suggestion
      this.iframe()
```

Also should we remove `delete this.iframe`?

Reply:
Is it needed? (`delete this.iframe`), I _nulled_ it (`this.iframe = null`)